app.py

```diff
@@ -55,8 +56,10 @@ def check_if_token_in_blocklist(jwt_header, decrypted_token):
     def format_unauthorized_message(message):
         return {app.config["JWT_ERROR_MESSAGE_KEY"]: message.capitalize()}, 401
 
-    logging.basicConfig()
-    logging.getLogger("sqlalchemy.engine").setLevel(logging.INFO)
+    if os.getenv("DB_LOGGING"):
```

Comment:
Would it be better to do something like:
```suggestion
    if os.getenv("DB_LOGGING") == "ON":
```

Just thinking that some people (namely me), might try to turn logging off by setting the env variable to "OFF"

Reply:
This is a fair point. I can definitely see people changing this to "OFF" later instead of commenting it out.

Reply:
Yes, thank you!